src/aio/tests/command.txt

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
What does this file used for?

Reply:
It's used for copy data from `command.txt` to `command.copy.txt` in `aio.dsn_file` test,. And the content in `command.txt` in meaningless.

Reply:
I recommend rename this file name and add comments in unit test to show what is the file used for and its content is meaningless.

Reply:
Yes, you are right. But it is not my main purpose in this pr. Maybe we can do it in later pull request.